src/content-tags-drawer/messages.js

```diff
@@ -33,6 +33,14 @@ const messages = defineMessages({
     id: 'course-authoring.content-tags-drawer.content-tags-collapsible.selectable-box.selection.aria.label',
     defaultMessage: 'taxonomy tags selection',
   },
+  manageTagsButton: {
+    id: 'course-authoring.content-tags-drawer.button.manage',
+    defaultMessage: 'Manage Tags',
+  },
+  tagsSidebarTitle: {
+    id: 'course-authoring.course-unit.sidebar.tags.title',
+    defaultMessage: 'Unit Tags',
```

Comment:
It would be awesome if you could add some descriptions to these new messages.  We'll be making them a requirement pretty soon.  See https://github.com/openedx/frontend-build/issues/517.

Reply:
Done https://github.com/openedx/frontend-app-course-authoring/pull/852/commits/c9261e49d8bcbf62675af3419b30309221b6bdf8